installer.tpl.sh

```diff
@@ -122,6 +131,11 @@ setup_type ()
 
 setup_ver ()
 {
+  if [ "${GC64:-OFF}" = "ON"  ] || [ arch = "aarch64" ]; then
```

Comment:
1. `arch = "aarch64"` is not correct. It should be `"${arch}" = "aarch64"` but I recommend using just `$(arch)`.
2. ARM packages go to the repo without the `gc64` suffix. Only `x86_64` packages go to the repo with the `gc64` suffix. So it should be something like this:
```
if [ "$(arch)" = "x86_64" ] && [ "${GC64:-OFF}" = "ON" ]; then
  echo "GC64 will be used"
  gc64="-gc64"
fi
```

Reply:
replaced with a correct conditional